.gitignore

```diff
@@ -73,3 +73,8 @@ package-lock.json
 ## Dist folders
 dist
 types
+
+# Module compiled output
+packages/core/src/*.js*
```

Comment:
I do not get that compiled on the branch.

Reply:
Hmm, i had these after running `npm run build`

Reply:
I think this might come from `tsc` in `packages/core`, maybe https://github.com/aerogear/aerogear-js-sdk/blob/master/packages/core/package.json#L13 should be `webpack` ?